scripts/lib/CIME/hist_utils.py

```diff
@@ -387,6 +387,20 @@ def get_extension(model, filepath):
 
     return result
 
+def generate_teststatus(testdir, baseline_dir):
```

Comment:
This function seems kind of out of place in hist_utils, but I'm willing to let it go if you want it here.